### PR TITLE
Adds documentation for community supported platforms

### PR DESCRIPTION
**Pull request by rishacha**
Add instructions for community maintained packages.

Ref : Github issue #5332

@jin Is this how you want it?

### CODE REVIEW

File: site/docs/install.md [hunk 1]
```diff
@@ -12,4 +12,14 @@ See the instructions for installing Bazel on:
 *   [macOS](install-os-x.md)
 *   [Windows](install-windows.md)
 
+## Community Supported Platforms
```
Comment (jin) [inline site/docs/install.md]: "Community Supported Packages"?

**Comment by jin** [comment]
@laurentlb @dslomov @spomorski PTAL at this PR? It adds a list of community supported packages to our install page.

File: site/docs/install.md (hunk 1 above)
Comment (spomorski) [reply site/docs/install.md]: Also, "Community-Supported".

Comment (spomorski) [reply site/docs/install.md]: Also, "Community-Supported".

Comment (rishacha) [reply site/docs/install.md]: Sorry, platforms is not the right word. I've fixed it to `Community-Supported Packages`

**Comment by jin** [comment]
thanks! importing.

**Pull request closed without merging**